packages/lit-dev-server/src/redirects.ts

```diff
@@ -97,12 +97,12 @@ export const oldLitElementSiteRedirects = new Map([
   ['/guide/build',      '/docs/v1/tools/build/'],
   ['/guide/community',  '/docs/v1/resources/community/'],
 
-  ['/try',              '/tutorials/'],
-  ['/try/create',       '/tutorials/'],
-  ['/try/properties',   '/tutorials/'],
-  ['/try/logic',        '/tutorials/'],
-  ['/try/events',       '/tutorials/'],
-  ['/try/style',        '/tutorials/'],
+  ['/try',              '/learn/'],
```

Comment:
`/try/` was the old playground/tutorial page. I think it should link to `learn/#filter=tutorial`

Reply:
Thank you! Done